src/test/java/UserLoginTest.java

```diff
@@ -1,34 +1,43 @@
+import com.opencart.driver.DriverRepository;
 import com.opencart.navigation.Navigation;
 import com.opencart.steps.*;
 import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 
 import static com.opencart.enums.URLs.BASE_URL;
 
 public class UserLoginTest extends BaseTest {
 
-    @Test(priority = 1)
+    @BeforeMethod
     public void registerUserWithValidParameters() {
```

Comment:
You have two tests with "registerUserWithValidParameters" name, this and one into UserRegisterTest  class

Reply:
In class UserLoginTest renamed registerUserWithValidParameters to registerUser and changed method logic.